### ISSUE TITLE
Request: React-Spreadsheet Component

### ISSUE THREAD

**Issue by ndroock1** [cljsjs/packages#551]
Description: A spreadsheet component ( highly customizable grid with editable cells ) built in ReactJS

Source: https://github.com/felixrieseberg/React-Spreadsheet-Component

Demo: http://felixrieseberg.github.io/React-Spreadsheet-Component/

**Comment by ndroock1** [comment]
Does this help in creating the externs ? http://jmmk.github.io/javascript-externs-generator/

**Comment by Deraen** [comment]
@ndroock1 Yes, that is the generator recommended by documentation: https://github.com/cljsjs/packages/wiki/Creating-Externs
